src/NServiceBus.Core/Audit/SendToAudit.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Audit
+{
+    using Transport;
+    using System.Collections.Generic;
+    using Pipeline;
+    using Routing;
+
+    /// <summary>
+    /// Default audit action that sends the audit message to the configured audit queue.
+    /// </summary>
+    public class SendToAudit : AuditAction
+    {
+        /// <summary>
+        /// Gets the message and routing strategies this audit operaation should result in.
+        /// </summary>
+        public override IEnumerable<(OutgoingMessage, RoutingStrategy)> GetRoutingData(IAuditContext context)
```

Comment:
Would it make sense to align the design to what we are building up with the recoverability seam? Namely, having this method return IRoutingContext and have an extension method that allows creating those contexts from the current audit context?

Reply:
Agreed, fixed now